source/.res/replace.txt

```diff
@@ -94,6 +94,7 @@
 .. |pmm-linux-metrics.service| replace:: :file:`pmm-linux-metrics-42000.service`
 .. |pmm-mysql-metrics.service| replace:: :file:`pmm-mysql-metrics-42002.service`
 .. |pmm-mysql-queries.service| replace:: :file:`pmm-mysql-queries-0.service`
+.. |pmm-platform| replace:: PMM Platform
```

Comment:
I wonder if there should be Percona Platform but not PMM Platform - @rnovikovP please note if it should be changed

Reply:
And the same point for all the PR

Reply:
yeah, this better to be Percona Platform   +  it's more communicating with PMM Server, so let's now move it to pmm server side from pmm client.  More clear documentation is to come.